src/Pages/Services/PageLayout/style.ts

```diff
@@ -129,3 +129,12 @@ export const PageReviews = styled.div`
     margin-top: 9px;
   }
 `;
+
+export const ContactLink = styled.a`
+color: #ad0d0d;
+text-decoration: none;
+
+&:hover{
+  color: #000;
+}
```

Comment:
The format is not applied. @Priyansh1825 I thought we have checked in place. How did Sanskar able to commit without formatting?

Reply:
Done